interface/ceed-vec.c

```diff
@@ -234,11 +268,17 @@ int CeedVectorDestroy(CeedVector *vec) {
   int ierr;
 
   if (!*vec || --(*vec)->refcount > 0) return 0;
+
+  if ((*vec) && ((*vec)->state % 2) == 1)
```

Comment:
I forgot to check this in the revision. The `*vec` part isn't needed here because it was already checked above.